main.js

```diff
@@ -0,0 +1,117 @@
+document.getElementById("search").addEventListener("click", function () {
+  const userSearch = document.getElementById("search-query").value;
+  console.log(userSearch);
+
+  if (userSearch === "") {
```

Comment:
same as the other project, this is not a good validation. Empty spaces make the api call